src/standalone/Soil/energy_hydrology.jl

```diff
@@ -1042,6 +1057,66 @@ function soil_turbulent_fluxes_at_a_point(
     )
 end
 
+"""
+    create_soil_albedo_vars(
+        path::String,
+        surface_space;
+        regridder_type = nothing,
+        regrider_kwargs = (),
+        file_reader_kwargs = (),
+        compose_function = identity,
+    )
+
+A convenience function to create the soil albedo variables given a path to the file containing
+the variables PAR_albedo_dry, NIR_albedo_dry, PAR_albedo_wet, NIR_albedo_wet.
+"""
+function create_soil_albedo_vars(
+    path::String,
+    surface_space;
+    regridder_type = nothing,
+    regridder_kwargs = (),
+    file_reader_kwargs = (),
+    compose_function = identity,
+)
+    PAR_albedo_dry = SpaceVaryingInput(
+        path,
+        "PAR_albedo_dry",
+        surface_space;
+        regridder_type,
+        regridder_kwargs,
+        file_reader_kwargs,
+        compose_function,
+    )
+    NIR_albedo_dry = SpaceVaryingInput(
+        path,
+        "NIR_albedo_dry",
+        surface_space;
+        regridder_type,
+        regridder_kwargs,
+        file_reader_kwargs,
+        compose_function,
+    )
+    PAR_albedo_wet = SpaceVaryingInput(
+        path,
+        "PAR_albedo_wet",
+        surface_space;
+        regridder_type,
+        regridder_kwargs,
+        file_reader_kwargs,
+        compose_function,
+    )
+    NIR_albedo_wet = SpaceVaryingInput(
+        path,
+        "NIR_albedo_wet",
+        surface_space;
+        regridder_type,
+        regridder_kwargs,
+        file_reader_kwargs,
+        compose_function,
+    )
+
+    return PAR_albedo_dry, NIR_albedo_dry, PAR_albedo_wet, NIR_albedo_wet
```

Comment:
You could write this as
```suggestion
    return map(
    s -> SpaceVaryingInput(
        path,
       s,
        surface_space;
        regridder_type,
        regridder_kwargs,
        file_reader_kwargs,
        compose_function,
    ), ("PAR_albedo_dry", "NIR_albedo_dry", "PAR_albedo_wet", "NIT_albedo_dry"))
```

I also think that we don't need to expose `compose_function` here. It's hard to see how it would be used.